source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst

```diff
@@ -220,6 +220,12 @@ Enabling camera sensors
     |__PART_FAMILY_NAME__| SK supports the following 15-pin FFC compatible
     camera modules with **OV5640** sensor:
 
+.. ifconfig:: CONFIG_part_variant in ('AM62PX')
+
+    Arducam V3Link (Fusion Mini) board can also be used to connect these camera modules to SK-AM62P over FPDLink cable.
```

Comment:
For all ifconfigs & code-block directives in this file ensure that it clearly lines up with directive.

Refer - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace